pyproject.toml

```diff
@@ -176,7 +177,6 @@ filterwarnings = [
     "error:The specified parameters:FutureWarning",
     # When calling `.show()` in tests, this is raised
     "ignore:FigureCanvasAgg is non-interactive:UserWarning",
-
```

Comment:
I put that blank line there intentionally, because you got confused one time and added a warning filter in the middle of the error block.